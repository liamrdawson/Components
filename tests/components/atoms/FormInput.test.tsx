import React from 'react'
import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { TextInput, TextArea } from '../../../src'

test('Snapshot Test: TextInput', () => {
  const {container} = render(
    <TextInput
      type="text"
      label="Text input label"
    />,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: TextArea', () => {
  const {container} = render(
    <TextArea
      label="Secondary Button"
    />,
  )
  expect(container).toMatchSnapshot()
  })