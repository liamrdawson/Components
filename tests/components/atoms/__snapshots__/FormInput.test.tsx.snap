// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: TextArea 1`] = `
<div>
  <div
    class="css-z21lfh-TextArea"
  >
    <label
      class="css-1bawu2g-TextArea"
      for="Secondary Button"
    >
      Secondary Button
    </label>
    <textarea
      class="css-1jf3zmb-TextArea"
      id="Secondary Button"
      name="Secondary Button"
    />
  </div>
</div>
`;

exports[`Snapshot Test: TextInput 1`] = `
<div>
  <div
    class="css-6r466p-TextInput"
  >
    <label
      class="css-2x82w0-TextInput"
      for="Text input label"
    >
      Text input label
    </label>
    <input
      class="css-18ek46i-TextInput"
      id="Text input label"
      name="Text input label"
      type="text"
    />
  </div>
</div>
`;
